mobify/vellum
# Fix up dependencies.

Fix dependency setup and update devDependencies to match new Adaptive 2.x standards. Fixes #84 

Status: **Ready for review**

Reviewers: **(GitHub @<name>s of all reviewers)**
Ticket: N/A
Linked PRs: none
## Changes
- Don’t commit bower_components (only project repos should commit this);
- Move bower.json from out of test/ to root, where it belongs;
- Remove unused dev deps (spline);
- Replace modularized-normalize-scss (devDependency) with mobify/level to match adaptivejs 2.0.
## How to test-drive this PR
- Check out this branch
- Run `grunt` (default task)
- Open test/index.html in a browser and verify everything looks the same as master
## Research resources
- none
## Feedback
